python/GafferSceneTest/InteractiveRenderTest.py

```diff
@@ -1541,6 +1541,166 @@ def testRendererContextVariable( self ):
 		image = IECoreImage.ImageDisplayDriver.storedImage( "myLovelySphereRenderedIn" + renderer )
 		self.assertTrue( isinstance( image, IECoreImage.ImagePrimitive ) )
 
+	def testLightFilters( self ) :
```

Comment:
We're getting this error from the 3Delight tests now :

```
ERROR: testLightFilters (GafferDelightTest.InteractiveDelightRenderTest.InteractiveDelightRenderTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/disk1/john/dev/build/gaffer/python/GafferSceneTest/InteractiveRenderTest.py", line 1548, in testLightFilters
    script["light"], colorPlug = self._createSpotLight()
AttributeError: 'InteractiveDelightRenderTest' object has no attribute '_createSpotLight'
```

Can you add `@unittest.skip()` to InteractiveDelightRenderTest, since we don't support light filters for 3Delight yet?

Reply:
15f15a3